web-wallet/src/style/dusk-components/tooltip.css

```diff
@@ -16,6 +16,7 @@
 
 .dusk-tooltip.dusk-tooltip-error {
 	background-color: var(--error-color);
+	color: white;
```

Comment:
You declared a var for this, but then you use a standard named colour: there is a bit of discrepancy.

My opinion is that we don't need variables in our colours for pure black or pure white, but I'll leave this to you.

Reply:
Correct @nortonandreev

Reply:
Done, @ascartabelli. I am now reusing the `--on-error-color` var and have made it white, instead of light grey.